cubecrit/sync.py

```diff
@@ -7,7 +7,7 @@
 from flask_apscheduler import APScheduler
 from sqlalchemy import Connection, text
 
-DATA_URL = "https://raw.githubusercontent.com/bobertoyin/cubecrit-data/main"
+DATA_URL = "https://raw.githubusercontent.com/bobertoyin/cubecrit-data/added-bio"
```

Comment:
Two things:

1. Can you create a PR to merge your `added-bio` branch into `main` in the `cubecrit-data` repo?
2. After merging the above, can you change `DATA_URL` back to "https://raw.githubusercontent.com/bobertoyin/cubecrit-data/main"?

After this PR, I'm going to write an issue for us to move the CSV files from `cubecrit-data` into `cubecrit`: having them in separate repos makes it awkward when trying to add new database columns.